yarn.lock

```diff
@@ -9846,6 +9870,11 @@ lru-cache@^6.0.0:
   resolved "https://registry.yarnpkg.com/luxon/-/luxon-3.0.3.tgz#573e65531efd3d92265feb640f02ba7a192e2388"
   integrity sha512-+EfHWnF+UT7GgTnq5zXg3ldnTKL2zdv7QJgsU5bjjpbH17E3qi/puMhQyJVYuCq+FRkogvB5WB6iVvUr+E4a7w==
 
+luxon@^3.0.2:
```

Comment:
This is a strange yarn decision to add a new version of luxon when the above version matcher would fall in line with it 🤔 
```
"luxon@^1.28.0 || ^2.0.0 || ^3.0.2":
```
Maybe yarn can't add `, luxon@^3.0.2` on it because it's version matcher with `||` clauses. 🤷